lib/import/Importer.js

```diff
@@ -76,7 +76,12 @@ export function importBpmnDiagram(diagram, definitions, bpmnDiagram) {
     }
 
     var mainDiagram = bpmnDiagram || definitions.diagrams[0];
-    canvas.setActivePlane(mainDiagram.plane.bpmnElement.id);
+
+    canvas.setRootElement(
```

Comment:
Question to you @marstamm: How do we currently import a process + drill down right away?

I guess we don't support case, but rather import a single diagram only when it is being passed to the bpmn-js instance via `#open`:

```
await bpmnJs.open(diagramXML, 'ID_OF_DIAGRAM');
```

Reply:
If we did support importing all + opening a single diagram, we'd need to change this line below :arrow_down:.

Reply:
No, we do not support it at this point. However, it is on our roadmap: https://github.com/bpmn-io/bpmn-js/issues/1546

Reply:
In this case my proposal is to make importing future proof already: https://github.com/bpmn-io/bpmn-js/pull/1551/commits/67f42f53492c416ebf282cf80e95bce5e3002d67